library/include/ck/library/host_tensor/host_reduce_util.hpp

```diff
@@ -245,7 +222,7 @@ binop_with_nan_check2(std::function<void(AccDataType&, AccDataType, bool&)> opRe
                       int& accuIndex,
```

Comment:
Is this tensor index? If so, ```index_t``` should be used.

Reply:
Yes, need correction